locales/base/translation.json

```diff
@@ -1251,6 +1251,14 @@
     "bestRate": "Best Rate",
     "header": "Select Payment Method"
   },
+  "fiatConnectLinkAccountScreen": {
+    "bankAccount": {
+      "header": "Link Bank Account",
+      "bodyTitle": "Enter Bank Account Details",
```

Comment:
I know this is what's in the copy, but if this is going to be a generic screen, would it be better to drop "Bank" from the copy, since it won't necessarily be a bank account?

Reply:
good question, thats why I nested these under "bankAccount" and we could have different copies per fiat account type. I'll confirm

Reply:
confirmed with Nitya that we'll have specific copies